CRM/EFT/BAO/EFT.php

```diff
@@ -389,8 +389,8 @@ public static function addTrxnChapterFund($fts, $cfParams) {
         $params = [
           "entity_id" => $ft,
           "entity_table" => "civicrm_financial_trxn",
-          "chapter" => CRM_Utils_Array::value('chapter_code_trxn', $cfParams),
-          "fund" => CRM_Utils_Array::value('fund_code_trxn', $cfParams),
+          "chapter" => CRM_Utils_Array::value('chapter_code_trxn', $cfParams, CRM_Utils_Array::value('chapter_code', $cfParams)),
```

Comment:
I'm not sure if this is a thing or if my test set up was wrong but would appreciate thoughts on this change @Edzelopez

Reply:
Yes, looks good.